### DISCUSSION TITLE
Does this Plugin still have benefits with current Shopare 6.6.4.0

### DISCUSSION THREAD

**Discussion by develth** [FriendsOfShopware/FroshPlatformThumbnailProcessor#137] [Q&A] [answered]
### Question

Hello,

within Shopware 6.6.4.0 the `Remote Thumbnail Configuration` was announced, what seems todo pretty the same as this Plugin. Or does this one have some benefits?

Thanks!
Thomas

**Comment by tinect** [comment, marked as answer]
Hey,

The feature doesn't replace the plugin because:

- It generates thumbnails for all files in the folder, regardless of whether they are images or not (keyword: `mediaCanHaveThumbnails` of `ThumbnailService`).
- The original images remain unaltered.

If this suits your needs, you could consider removing the plugin.

Best Regards,
Sebastian

**Reply by yunits** [reply]
I would like to ask a question:
_The original images remain unaltered._
regarding to https://developer.shopware.com/docs/guides/plugins/plugins/content/media/remote-thumbnail-generation.html
I can't read anything that the images are changed or something similar. Could you clarify this point?

_It generates thumbnails for all files in the folder_
What kind of files could also have a thumbnail, except images? Do you mean videos? Or is there something I am missing?

**Reply by tinect** [reply]
> I can't read anything that the images are changed or something similar. Could you clarify this point?

Just the thumbnail-urls are handled, but not the original image, which is failover image - additionally to the size above the highest specified thumbnail size. 

> What kind of files could also have a thumbnail, except images? Do you mean videos? Or is there something I am missing?

Yes, this is applied to any media in the folder.

**Reply by yunits** [reply]
> Just the thumbnail-urls are handled, but not the original image, which is failover image - additionally to the size above the highest specified thumbnail size.

If I am not wrong this plugin adds to the original image (failover image) the specified parameter, for example "?width=1920".
Shopware doesn't do that, is that what you mean?

**Comment by develth** [comment]
Thanks for your fast Reply & Support, Sebastian!

**Comment by tinect** [comment]
I’d like to keep this issue/question open, as it might come up more frequently. Additionally, I've pinned it.
